### PR TITLE
Implementation of frequency table UDF

### PR DESCRIPTION
**Pull request by kcibul**
Construct a frequency table (histogram) of an array of elements.
Frequency table is represented as an array of STRUCT(value, freq)

### CODE REVIEW

File: udfs/community/freq_table.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION fn.freq_table(arr ANY TYPE) AS ((
+  SELECT 
+    STRUCT(x as value, COUNT(1) as freq) as entry
+  FROM UNNEST( arr ) AS x
+  GROUP BY x
+  ORDER BY x
+));
```
Comment (ryanmcdowell) [inline udfs/community/freq_table.sql]: Add terminating new-line

File: udfs/community/freq_table.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION fn.freq_table(arr ANY TYPE) AS ((
+  SELECT 
+    STRUCT(x as value, COUNT(1) as freq) as entry
+  FROM UNNEST( arr ) AS x
```
Comment (ryanmcdowell) [inline udfs/community/freq_table.sql]: Remove additional spaces around `arr`

File: udfs/community/freq_table.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (ryanmcdowell) [inline udfs/community/freq_table.sql]: `2020`

File: udfs/community/freq_table.sql (hunk 1 above)
Comment (kcibul) [reply udfs/community/freq_table.sql]: done

File: udfs/community/freq_table.sql (hunk 2 above)
Comment (kcibul) [reply udfs/community/freq_table.sql]: done

File: udfs/community/freq_table.sql (hunk 3 above)
Comment (kcibul) [reply udfs/community/freq_table.sql]: done

**Pull request merged**